src/data/reducers/table.js

```diff
@@ -52,6 +53,13 @@ const tableReducer = (state = {}, action) => {
         loading: false,
         error: action.payload.error,
       });
+    case CLEAR_TABLE:
+      return updateTable(state, action.payload.tableId, {
+        loading: false,
+        error: null,
+        ordering: undefined,
+        data: undefined,
```

Comment:
We've typically been using `null` as the default value when no data exists in our reducers. Both `ordering` and `data` could be set to `null` here. The [ordering value](https://github.com/edx/edx-portal/blob/master/src/data/actions/table.js#L87) in `actions/table.js` would ultimately get set to `undefined` regardless.